src/main/java/github/weichware10/toolbox/ConfirmBox.java

```diff
@@ -0,0 +1,57 @@
+package github.weichware10.toolbox;
+
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+
+//How to Communicate between windows
+public class ConfirmBox {
+
+    static boolean answer;
+
+    public static boolean display(String title, String message){
+        final Stage window = new Stage();
+
+        window.initModality(Modality.APPLICATION_MODAL);
+        window.setTitle(title);
+        window.setMinWidth(250);
+
+        Label label1 = new Label();
+        label1.setText(message);
+        
+        Button yesButton = new Button("Yes");
+        Button noButton = new Button("No");
+        yesButton.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.